packages/code-du-travail-frontend/src/outils/CommonIndemniteDepart/steps/Resultat/components/DecryptResult.tsx

```diff
@@ -6,21 +6,23 @@ type Props = {
   isAgreementSupported: boolean;
   legalResult: string;
   agreementResult?: string;
+  label: string;
```

Comment:
Je pense on aurait pu le typer fort en `licenciement` | `rupture`

```suggestion
  label: 'licenciement' | 'rupture conventionnelle'
```

Reply:
Ou alors reprendre le type : `IndemniteDepartType` que t'utilises à d'autre endroit